Containerfile

```diff
@@ -29,11 +20,18 @@ LABEL \
   com.redhat.component="build-trusted-artifacts"
 
 COPY --from=files / /
-COPY --from=oras /usr/local/bin/oras /usr/local/bin/
 
 RUN microdnf update --assumeyes --nodocs --setopt=keepcache=0 && \
-  microdnf install --assumeyes --nodocs --setopt=keepcache=0 tar gzip sysstat time jq && \
-  useradd --non-unique --uid 0 --gid 0 --shell /bin/bash notroot
+    microdnf install --assumeyes --nodocs --setopt=keepcache=0 tar gzip time jq && \
+    useradd --non-unique --uid 0 --gid 0 --shell /bin/bash notroot
+
+ARG ORAS_VERSION=1.1.0
+
+RUN curl -LO "https://github.com/oras-project/oras/releases/download/v${ORAS_VERSION}/oras_${ORAS_VERSION}_linux_amd64.tar.gz" && \
+    mkdir -p oras-install/ && \
+    tar -zxf oras_${ORAS_VERSION}_*.tar.gz -C oras-install/ && \
+    mv oras-install/oras /usr/local/bin/ && \
+    rm -rf oras_${ORAS_VERSION}_*.tar.gz oras-install/
```

Comment:
I find it a good idea to always run the command just installed just to make sure it is working as expected, i.e. add `&& oras version` to the end.